packages/gatsby/content/advanced/contributing.md

```diff
@@ -10,6 +10,18 @@ Thanks for being here! Yarn gives a lot of importance to being a community proje
 
 We have some rules regarding our issues. Please check [the following page](/advanced/sherlock) for more details.
 
+## How can you help?
+
+  - Review our documentation! We often aren't native english speakers, and our grammar might be a bit off. Any help we can get that makes our documentation more digest is appreciated!
+
+  - Talk about Yarn in your local meetups! Even our users aren't always aware of some of our features. Learn, then share your knowledge to your own circles!
+
+  - Help with our infra! There are always small improvements to do: run tests faster, uniformize the test names, improve the way our version numbers are setup, ...
+
+  - Write code! We have so many features we want to implement, and so little time to actually do it... Any help you can afford will be appreciated, and you will have the satisfaction to know that your work helped literally millions of developers!
+
+  - Enable the [monitoring plugin](), which gives us a better idea of how Yarn is used and which errors are the most frequent. You'll even get rewarded for it since you'll be made aware of any new release we can send your way!
```

Comment:
This isn't ready, but I'd like to eventually (maybe 2.1?) have an opt-in plugin (ie people would need to explicitly install it) that would track which commands are the most used, which errors are the most frequent, which versions are the most used, etc.